jadx-gui/src/main/java/jadx/gui/treemodel/JResource.java

```diff
@@ -200,6 +200,7 @@ private ICodeInfo loadContent() {
 			List<JResource> nodes = ResTableHelper.buildTree(rc);
 			sortResNodes(nodes);
 			subNodes = nodes;
+			update();
```

Comment:
```suggestion
			UiUtils.uiRun(this::update);
```
`loadContent` method might be called from a background thread.